src/RestApiContext.php

```diff
@@ -114,10 +127,31 @@ private function getRequest()
     }
 
     /**
-     * @return array|\Guzzle\Http\Message\Response
+     * @param array $headers
+     * @return string
      */
-    private function getResponse()
+    private function getRawHeaders(array $headers)
     {
-        return $this->restApiBrowser->getResponse();
+        $rawHeaders = '';
+        foreach ($headers as $key => $value) {
+            $rawHeaders .= sprintf("%s: %s\n", $key, is_array($value) ? implode(", ", $value) : $value);
+
+        }
+        $rawHeaders .= "\n";
+        return $rawHeaders;
+    }
+
+    /**
+     * @Then print response
+     */
+    public function printResponse()
```

Comment:
printing only response does not mean anything for me. 
We can add `print request` step np if you have some need.  But from our side we need to keep the current behavior of `print response`.